ui/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "main": "nuxeo-ui-elements.js",
   "dependencies": {
-    "@nuxeo/nuxeo-elements": "^3.0.0-SNAPSHOT",
+    "@nuxeo/nuxeo-elements": "3.0.0-SNAPSHOT",
     "@nuxeo/paper-typeahead": "^0.6.0-SNAPSHOT",
```

Comment:
Shouldn't we remove the caret here too?

Reply:
Removing the caret is only necessary for packages that are being redeployed with the same version.